### ISSUE TITLE
Seeking in libVLC player sets time to beginning

### ISSUE THREAD

**Issue by brianclinkenbeard** [jellyfin/jellyfin-androidtv#544]
When seeking forward or back while playing a video in the libVLC player, the A/V goes to the right time but then the time shown restarts back at 0:00. This interferes with subtitles as well. 

Problem occurs on an MKV file with embedded subtitles.

**Comment by brianclinkenbeard** [comment]
This appears to be a duplicate of #400.

**Comment by ManuLinares** [comment]
> This appears to be a duplicate of #400.

Sorry, it's not.
 
Tried master branch, and 11.5. issue persists

In libVLC, when seeking, the video seek is fine, sometimes, but it always sets the time at zero. So subtitles are desynced as they start again.

**Comment by ManuLinares** [comment]
https://github.com/jellyfin/jellyfin-androidtv/commit/42902365172bd4fbe7d5ee60b48161d3af260484#r50257236

**Comment by brianclinkenbeard** [comment]
> Sorry, it's not.

You're correct, I intended to reopen this issue but forgot about it. The ExoPlayer provided in the current version has its own issues, and combined with this issue it renders subtitles impossible for many files.

**Comment by ClockeNessMnstr** [comment]
Any updates on this?

**Comment by nielsvanvelzen** [comment]
It might be fixed in the 0.12 beta, please try it out and let us know

**Comment by ClockeNessMnstr** [comment]
Thanks, I'll give it a shot. Also realized I could use Kodi. Seems to work well with the jellyfin add on

**Comment by ClockeNessMnstr** [comment]
Still broken

**Comment by mueslimak3r** [comment]
likely fixed by #1251

**Comment by mueslimak3r** [comment]
> I can confirm, seeking is working as intended 👍 
> 
> Weird, with libVLC the seeking timer jumps 2 seconds instead of 1 often. This causes some subtitles to not show. (skips some subtitle lines). I will open a new issue for this right?

Is that with the stable 0.12.3 release on the app store, or is that with a dev build?

And when you say seek timer, is that while/after seeking? Like pressing the ff/rw buttons on your remote or in the UI?

**Comment by ManuLinares** [comment]
Latest Master Dev build app, Jellyfin 10.7.7 server, using libVLC. (Exoplayer works perfect)

Not when seeking, always.
Description: When playing *any* video, the seekbar timer skips every few seconds, a second. Ex: goes from 00:00:05 to 00:00:07.

This causes subtitles to skip lines (don't show lines in that skipped second)

**Comment by mueslimak3r** [comment]
Is that while transcoding or direct playing/streaming?

**Comment by ManuLinares** [comment]
Transcoded videos only. And only after seeking or starting a video with "Resume from x:xx".

It seems if I don't seek, or "video direct" it works without skipping seconds in the timer.
Also worth mentioning that video image plays fine, it's only the timer, and subsequentaly the subtitles

**Comment by mueslimak3r** [comment]
@ManuLinares I'll look into that. Can you open a new issue for this so we can track it and have a dedicated discussion there?

**Comment by ManuLinares** [comment]
No problem, I created a new issue https://github.com/jellyfin/jellyfin-androidtv/issues/1347
